libecho/include/echo.h

```diff
@@ -9,13 +9,16 @@
 #include <vector>
 
 namespace echo {
```

Comment:
mam wrażenie, że lepiej by było upakować te dwie funkcje w klasę; wtedy zamiast initEcho mógłby być konstruktor

Reply:
Akurat initEcho musi być wywołane tylko raz na program (jeśli nie używamy QT, jeśli uzywamy to wcale nie trzeba), i jeśli uzywamy QT w programie korzystającym z naszej libki, to nie może być wywołane przed stworzeniem QApplication. W tym PR nie będę tego zmieniał, ale @Riyria-was-taken zmieni to w swojej części, że to faktycznie będzie klasa, a initEcho to statyczna metoda.